index.html

```diff
@@ -234,5 +234,63 @@ <h2>Contact me</h2>
      </div>
     </footer> 
     <script src="script.js"></script> 
+    <div class="container-modal">
+      <div class="inner-container">
+      <div class = "main-container-head">
+      <h2>Tonic</h2>
+      <img class = "popup-cross" src="./images/Enabled-3.svg" alt="X">
+      </div>
+      <div class="main-container-body">
+        <img src="./images/frame2.svg" alt="frame"> <br> 
+        <img class = "main-project-img-mobile" src="./images/SnapshootPortfolio.svg" alt="main-img"> <br> <br>
+        <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.
+           Non tenetur quaerat autem odit alias, dignissimos nulla 
+           doloribus at, quae perferendis delectus sapiente officiis
+           eius molestiae esse aliquam, dicta illum quidem! Lorem ipsum
+           dolor sit amet consectetur adipisicing elit. Dolore quos corrupti 
+           et illum reiciendis quisquam alias iure amet omnis, excepturi odio 
+           eligendi dolorem itaque fugiat delectus, minus explicabo inventore 
+           aliquid!</p> <br>
+           <img src="./images/Tags.svg" alt="">
+      </div>
+      <div class="popup-button-container">
+      <button class="project-popup-button-1">See Live</button>
+      <button class="project-popup-button-2">See Source</button>
+    </div>
+    </div>
+  </div>
+  <div class="container-desktop-modal">
+    <div class="inner-desktop-container">
+      <div class="main-desktop-head-container">
+        <h3>Tonic</h3>
+        <img class = "popup-cross-desktop" src="./images/Enabled-4.svg" alt="X">
+      </div>
+      <div class="main-container-body-desktop">
+        <img class = "skill-list-img" src="./images/frame2.svg" alt="frame"> <br>
+        <img src="./images/projrctDetails-snap.svg" alt="main-img"><br> <br>
+        <div class="flex-container-desktop">
+          <div class="para-container-desktop-popup">
+            <p class="popUpWindowPara">Lorem ipsum dolor sit amet, consectetur adipisicing elit. 
+              Ad dignissimos minus quae consectetur eius, perferendis quas 
+              recusandae perspiciatis veniam quaerat, qui eaque quam tenetur 
+              quo ipsam commodi porro officia modi?Lorem ipsum, dolor sit amet 
+              consectetur adipisicing elit. Cum ad minus culpa pariatur veniam animi.
+              Est asperiores molestias illum beatae eius, consequuntur hic quas aperiam
+              corporis quisquam deleniti repellendus esse?</p>
+            </div>
+            <div class="side-bar">
+              <div class="skill-tag-img-container">
+                <img src="./images/Tags.svg" alt="img">
+                <img src="./images/Tags-triple.svg" alt="img">
+              </div>
+              <div class="button-desktop-popup-flex">
+                <button>See Live</button>
+                <button>See Source</button>
```

Comment:
- [ ] Your design is almost the same as provided in Figma design. However, you still need to add `icons` in these buttons `See Live` and `See Source`. Take a look at the below screenshots, please 👇 
- Your design 👇 
![Screenshot (220)_LI](https://user-images.githubusercontent.com/67087664/156514761-377a193e-ff9b-4240-bbc2-e63f3860a32c.jpg)
 - Figma design 👇 
![Screenshot (214)](https://user-images.githubusercontent.com/67087664/156514872-0cd06309-b2e2-4fbd-af36-9884e4c61313.png)